hieradata_aws/common.yaml

```diff
@@ -1022,9 +1022,9 @@ grafana::dashboards::application_dashboards:
   feedback: {}
   finder-frontend:
     rows:
-      - content_store_request_time
-      - registry_request_time
-      - search_api_request_time
+      - - content_store_request_time
+        - registry_request_time
+        - search_api_request_time
```

Comment:
I don't really know YAML, but if this is a fix of some kind, it might be clearer to squash it with the previous commit.

Reply:
Thanks - I squashed the commits.